pkg/ocmagenthandler/ocmagenthandler_networkpolicy.go

```diff
@@ -26,7 +26,9 @@ func buildNetworkPolicy(ocmAgent ocmagentv1alpha1.OcmAgent) netv1.NetworkPolicy
 			MatchExpressions: []metav1.LabelSelectorRequirement{{
```

Comment:
Based on the test in a staging HCP cluster, this FleetMode networkpolicy doesn't work to allow the ingress traffic from `openshift-observability-operator` namespace which means the `ocm-agent-fleet` cannot receive alerts payload from OBO alertmanager in that `openshift-observability-operator` namespace.

We might need to update the fleetmode networkpolicy as below: 
```
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: ocm-agent-allow-obo-alertmanager
  namespace: openshift-ocm-agent-operator
spec:
  ingress:
  - from:
    - namespaceSelector:
        matchLabels:
          kubernetes.io/metadata.name: openshift-observability-operator <<<<<<<<<<<<<<<<<
  podSelector:
    matchLabels:
      app: ocm-agent-fleet
  policyTypes:
  - Ingress
```
Basically, to use the matchLabels [kubernetes.io/metadata.name](http://kubernetes.io/metadata.name) to allow the ingress traffic from `openshift-observability-operato`r namespace.